brave/brave-core
# feat(wallet): Hide NFTs Tab Setting in Portfolio Settings

## Description 
Adds a hide `NFTs tab` check box to the `Portfolio Settings` menu


Resolves <https://github.com/brave/brave-browser/issues/30345>



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Go to the `Portfolio` page and open the `Portfolio Settings` menu
2. Uncheck the `NFTs tab` checkbox
3. The segmented controls on the `Portfolio Overview` page
4. Refresh the page, the selection should be `persisted`
5. Switch the `NFTs tab` back on and go to the `NFTs` tab
6. Uncheck the `NFTs tab` again, it should route back to the `Assets` page.

https://github.com/brave/brave-core/assets/40611140/2a4c8624-d783-433a-8e8a-ba102f8f664c
